src/plugins/cordova-plugin-device/sim-host.js

```diff
@@ -312,38 +388,66 @@ function handleSelectDevice() {
     var deviceList = document.getElementById('device-list');
     var option = deviceList.options[deviceList.selectedIndex];
     document.getElementById('device-model').value = option.getAttribute('_model');
+    document.getElementById('device-manufacturer').value = option.getAttribute('_manufacturer');
     document.getElementById('device-platform').value = option.getAttribute('_platform');
     document.getElementById('device-uuid').value = option.getAttribute('_uuid');
     document.getElementById('device-version').value = option.getAttribute('_version');
+    document.getElementById('is-virtual-device').checked = true; // by default, true for all devices
+    document.getElementById('device-serial').value = '123456789';
+}
+
+function registerTelemetryForControl(controlId) {
+    document.getElementById(controlId).addEventListener('change', telemetry.sendUITelemetry.bind(this, Object.assign({}, baseProps, {
+        control: controlId
+    })));
 }
 
 function registerTelemetryEvents() {
     // Register the simple events (onchange -> send the control ID).
     var basicTelemetryEventControls = [
         'device-model',
+        'device-manufacturer',
         'device-platform',
         'device-uuid',
-        'device-version'
+        'device-version',
+        'device-serial'
     ];
 
     basicTelemetryEventControls.forEach(function (controlId) {
         registerTelemetryForControl(controlId);
     });
 
-    // Register the event for the tdevice combo box.
+    // Register the event for the device combo box.
     var deviceList = document.getElementById('device-list');
 
-    deviceList.onchange = function () {
+    deviceList.addEventListener('change', function () {
         var option = deviceList.options[deviceList.selectedIndex];
```

Comment:
@guillaumejenkins @dlebu This new event listener for the change event added as part of the telemetry feature, overwrites the previous event listener added using deviceList.onchange, to update the fields according to the device selected. I noticed that when I've updated my branch,  so I fixed here by replacing the usages of onchange with addEventListener, here for the telemetry and for when the device is selected.